spec/models/collection_spec.rb

```diff
@@ -25,11 +25,29 @@
       it { is_expected.to eq 0 }
     end
 
-    context "with two 50 byte files" do
-      let(:bitstream) { double("content", size: "50")}
+    context "with three 33 byte files" do
+      let(:bitstream) { double("content", size: "33")}
       let(:file) { mock_model GenericFile, content: bitstream }
-      before { allow(@collection).to receive(:members).and_return([file, file]) }
-      it { is_expected.to eq 100 }
+      let(:documents) { 
+        [{ 'id' => 'file-1', 'file_size_is' => 33 }, 
+         { 'id' => 'file-2', 'file_size_is' => 33 }, 
+         { 'id' => 'file-3', 'file_size_is' => 33 }
+        ]}
+
+      before { 
+        allow(@collection).to receive(:members).and_return([file, file, file])
+        allow(ActiveFedora::SolrService).to receive(:query).with(
+            "_query_:\"{!raw f=has_model_ssim}GenericFile\"",
```

Comment:
Should you be using `construct_query_for_rel` here for parallelism with the code below?